index.js

```diff
@@ -133,15 +150,14 @@ module.exports = {
             }
         };
 
-        //Adding event listeners to take automatic screenshot
+        // Adding event listeners to take automatic screenshot
         if (autoCaptureOptions.indexOf('click') !== -1) {
             flow.on('scheduleTask', function (task) {
```

Comment:
Not exactly sure, could be a bug with Github or it could be expected if the changes are not affecting the lines that have updated in develop.